src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -112,6 +111,11 @@
         return reportFoldersAndFiles;
     }
 
+    private static void prepareTemplateFile(String outputPath) throws IOException {
```

Comment:
@ccyccyccy, perhaps this method can be documented using Javadoc?

Reply:
Thanks for the reminder. I forgot about it.